### PR TITLE
Added netlify logo to README and Footer

### PR DESCRIPTION
**Pull request by ChrisHinchey**
Signed-off-by: Christopher Hinchey <chinchey@mitre.org>

### CODE REVIEW

**Comment by aaronlippold** [comment]
Let’s see if we can resize about half as large
<details><summary>Browser metadata</summary>





```
Path:      /#/
Browser:   Safari 15.6 on Mac OS 10.15.6
Viewport:  834 x 1151 @2x
Language:  en-US
Cookies:   Disabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-395--mitre-saf.netlify.app%2F%23%2F&os=OS%20X&os_version=Catalina&browser=Safari&browser_version=15.0&resolution=834x1151&speed=1&start=true&ref=netlify-source)



</details>

[Open Deploy Preview](https://deploy-preview-395--mitre-saf.netlify.app/#/) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=63376a9dc2e1140009a29c48&commentID=63384d77478ad57d14faa20f&resolution=resolved)

**Comment by em-c-rod** [comment]
LGTM :+1:

File: src/components/core/Footer.vue [hunk 1]
```diff
@@ -122,6 +129,15 @@ export default {
       FooterData: FooterData,
     };
   },
+  computed : {
+    imgVersion()  {
+      if (this.$vuetify.theme.dark) {
+        return '-light';
```
Comment (em-c-rod) [inline src/components/core/Footer.vue:135]: Actually, I didn't see the "light" picture show up in the preview. Am I missing something?

Comment (ChrisHinchey) [reply src/components/core/Footer.vue:135]: There should be a different picture for both light and dark mode. It is in the footer on the left side.

Comment (em-c-rod) [reply src/components/core/Footer.vue:135]: It looks the same to me when I toggle the light to dark mode on the preview.

Comment (ChrisHinchey) [reply src/components/core/Footer.vue:68]: I made some new changes, so hopefully when it finishes building it will fix the problem.

**Pull request merged**